app.js

```diff
@@ -3,129 +3,185 @@
 //correct keeps track of user's correct answers.
 var correct = 0;
 
-var confirmed = confirm('Are you ready to rumble? Note: All questions must be answered with y or n.');
-
-if (confirmed === true) {
-  console.log('The user is rumblin\'');
-  alert('Let\'s begin!');
-} else {
-  console.log('User is not rumblin\' but too bad, we\'re doing it anyway. \:\)');
-  alert('Too bad, we\'re doing it anyway.');
+function funct(){
+  var confirmed = confirm('Are you ready to rumble? Note: All questions must be answered with y or n.');
+  if (confirmed === true) {
+    console.log('The user is rumblin\'');
+    alert('Let\'s begin!');
+  } else {
+    console.log('User is not rumblin\' but too bad, we\'re doing it anyway. \:\)');
+    alert('Too bad, we\'re doing it anyway.');
+  }
 }
 
-var answer1 = prompt('In a standup fight, would you win against this developer?');
-if (answer1.toUpperCase() === 'Y') {
+function funct1() {
+  var answer1 = prompt('In a standup fight, would you win against this developer?');
+  answer1 = answer1.slice(0, 1);
```

Comment:
solid approach, you could also do something like `if (answer1[0].toUpperCase() === 'Y')` and save yourself a line of code.